script/translations.js

```diff
@@ -0,0 +1,18 @@
+window.translations = {
+  "Closed": {{ translateJS phrase="Closed" context="Indicates that a store is closed" }},
+  "Open 24 Hours": {{ translateJS phrase="Open 24 Hours" context="The hours that a store is open" }},
+  "Opens at": {{ translateJS phrase="Opens at" context="Indicates the time a store will open" }},
+  "Open Now": {{ translateJS phrase="Open Now" context="Indicates that a store is open" }},
+  "Closes at": {{ translateJS phrase="Closes at" context="Indicates the time a store will close" }},
+  "Day of the Week": {{ translateJS phrase="Day of the Week"}},
+  "Hours": {{ translateJS phrase="Hours"}},
+  "Yes": {{ translateJS phrase="Yes" }},
+  "No": {{ translateJS phrase="No" }},
+  MONDAY: {{ translateJS phrase="Monday" }},
```

Comment:
Would it make sense to make the days of the week keys equal to the phrase too? (like same MONDAY -> Monday to match the phrase)

Reply:
the keys used here match with the text pass to `stringlocalizer.js` getTranslation(text). In this case the week days are all passed in as all capitalized. We would have to do some string transformation inside getTranslation() if we want to change this to match the phrase

Reply:
ok sgtm!